docs/core-features/extensions.md

```diff
@@ -374,25 +373,18 @@ assert circuit.encrypt_run_decrypt(-1) == 0
 assert circuit.encrypt_run_decrypt(-3) == 0
 assert circuit.encrypt_run_decrypt(5) == 5
 ```
+### ReLU Conversion methods
+The ReLU operation can be implemented in two ways:
+- **Single TLU (Table Lookup Unit) on the original bit-width**: Suitable for small bit-widths, as it requires fewer resources.
+- **Multiple TLUs on smaller bit-widths**: Better for large bit-widths, avoiding the high cost of a single large TLU.
 
-ReLU extension can be converted in two different ways:
-
-* With a single TLU on the original bit-width.
-* With multiple TLUs on smaller bit-widths.
-
-For small bit-widths, the first one is better as it'll have a single TLU on a small bit-width. For big bit-widths, the second one is better as it won't have a TLU on a big bit-width.
+### Configuration options
 
-The decision between the two can be controlled with `relu_on_bits_threshold: int = 7` configuration option:
+The method of conversion is controlled by the `relu_on_bits_threshold: int = 7` option. For example, setting `relu_on_bits_threshold=5` means:
```

Comment:
I think it's `relu_on_bits_threshold: int = 5` here, to be confirmed by @umut-sahin

Reply:
indeed ok